### PR TITLE
fix(ui-ux): styling

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What this PR does / why we need it:
- "Error guide" underline styling: TBC for actual link
image

- Spacing of Getting started's paragraphs
image

-Initiate green line to be consistent with figma
image

- Fix Review ERC20 tx address width text overflow 
image

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/erc-transfer/StepThreeVerification.tsx [hunk 1]
```diff
@@ -46,7 +45,18 @@ export default function StepThreeVerification({
   const [title, setTitle] = useState<TitleLabel | RejectedLabelType>(
     TitleLabel.Validating
   );
-  const [content, setContent] = useState<ContentLabel>(ContentLabel.Rejected);
+  const contentLabelRejected = (
+    <div>
```
Comment (pierregee) [inline apps/web/src/components/erc-transfer/StepThreeVerification.tsx:49]: image

File: apps/web/src/components/erc-transfer/StepOneInitiate.tsx [hunk 2]
```diff
@@ -33,8 +33,8 @@ export default function StepOneInitiate({
           details are correct and funds are sent in a single transaction, with a
           stable network connection.
         </p>
-        <div>
-          <div className={clsx("inline text-sm text-dark-900 mt-1", "md:mt-2")}>
+        <div className="pt-4">
```
[the remark this answers is not in the document]
Comment (chloezxyy) [reply apps/web/src/components/erc-transfer/StepOneInitiate.tsx:36]: What px size is your screen on? Looks fine on my side 
image

File: apps/web/src/components/erc-transfer/StepThreeVerification.tsx (hunk 1 above)
Comment (chloezxyy) [reply apps/web/src/components/erc-transfer/StepThreeVerification.tsx]: Resolved

File: apps/web/src/components/erc-transfer/StepOneInitiate.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pierregee) [reply apps/web/src/components/erc-transfer/StepOneInitiate.tsx:36]: can't replicate it again!

**Review by pierregee — APPROVED** [review]

**Pull request merged**